### PR TITLE
Minor doc updates.

### PR DESCRIPTION
**Pull request by bangerth**

### CODE REVIEW

File: include/deal.II/base/function_lib.h [hunk 1]
```diff
@@ -1482,6 +1482,13 @@ namespace Functions
      * separately. In other words, there is no need to keep the original object
      * from which this object could copy its information, but it might as well
      * take over ("move") the data.
+     *
+     * Moving data also enables using tables that are located in shared memory
+     * between multiple MPI processes, rather than copying the data from
+     * shared memory into local memory whenever one creates an
+     * InterpolatedTensorProductGridData object. See the
+     * TableBase::replicate_across_communicator() function on how to shared a
```
Comment (kronbichler) [inline include/deal.II/base/function_lib.h:1490]:
```suggestion
     * TableBase::replicate_across_communicator() function on how to share a
```

File: include/deal.II/base/function_lib.h [hunk 2]
```diff
@@ -1620,6 +1627,13 @@ namespace Functions
      * separately. In other words, there is no need to keep the original object
      * from which this object could copy its information, but it might as well
      * take over ("move") the data.
+     *
+     * Moving data also enables using tables that are located in shared memory
+     * between multiple MPI processes, rather than copying the data from
+     * shared memory into local memory whenever one creates an
+     * InterpolatedUniformGridData object. See the
+     * TableBase::replicate_across_communicator() function on how to shared a
```
Comment (kronbichler) [inline include/deal.II/base/function_lib.h:1635]:
```suggestion
     * TableBase::replicate_across_communicator() function on how to share a
```

**Review by kronbichler — APPROVED** [review]
OK apart from two typos.

**Comment by bangerth** [comment]
Thanks, @kronbichler . I fixed the typos.

**Review by kronbichler — APPROVED** [review]

**Pull request merged**